conans/client/installer.py

```diff
@@ -327,6 +318,25 @@ def _handle_node_editable(self, node):
                                        settings=node.conanfile.settings,
                                        options=node.conanfile.options)
 
+            build_folder = editable_cpp_info.folder(node.ref, "build_folder",
```

Comment:
Is it worth to add constants for `build_folder` adn `source_folder` in the `EditableLayout` class?